include/blogWithComments_functions.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+$Error = array();
+
+
+function RepopulateComments($info){
```

Comment:
This function works, but it could be even more general. Right now it only works with fields in the `submitComment` form. Can you think of a way to make it so it'd work with any form?